output/mapping/F23_2014.csv

```diff
@@ -128,11 +128,11 @@ xpath,label-key,index,comment,guidance
 /AWARD_CONTRACT/AWARDED_CONTRACT/VALUES/VAL_ESTIMATED_TOTAL/@CURRENCY,currency,,,"For the *Initial estimated total value of the contract/lot*, discard as above. For the *Total value of the contract/lot*, map to the contract's `.value.currency`."
 /AWARD_CONTRACT/AWARDED_CONTRACT/VALUES/VAL_TOTAL,value_total_final_concess,,,Map to the contract's `.value.amount`
 /AWARD_CONTRACT/AWARDED_CONTRACT/VALUES/VAL_TOTAL/@CURRENCY,currency,,,/AWARD_CONTRACT/AWARDED_CONTRACT/VALUES/VAL_ESTIMATED_TOTAL/@CURRENCY
-/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_REVENUE,concess_fees_prices,,,
-/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_REVENUE/@CURRENCY,,,,
-/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_PRICE_PAYMENT,concess_payments,,,
-/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_PRICE_PAYMENT/@CURRENCY,,,,
-/AWARD_CONTRACT/AWARDED_CONTRACT/INFO_ADD_VALUE,concess_other_details,,,
+/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_REVENUE,concess_fees_prices,,,Map to `award.concession.revenueFromFees.amount`.
+/AWARD_CONTRACT/AWARDED_CONTRACT/VAL_REVENUE/@CURRENCY,,,,Map to `award.concession.revenueFromFees.currency`.
```

Comment:
The PPP extension adds a [`contracts.financeSummary`](https://github.com/open-contracting-extensions/ocds_ppp_extension/blob/master/release-schema.json#L151) field.

I think these two values are similar 'summary' statistics about the financing of the concession (cc @duncandewhurst). It might therefore be appropriate to re-use a `financeSummary` scope instead of making these fields specific with `concession`.

@duncandewhurst Let us know if you have other suggestions in terms of structure or field names.

Reply:
If I understood the meaning of the fields correctly (the total expected revenue from user payments and government payments during the course of the concession) then I think it would be more appropriate to use the [charges extension](https://extensions.open-contracting.org/en/extensions/charges/master/):

> used to record details of the total charges that are estimated or applied to users or government during the operation of a Public Private Partnership contract.

The finance summary is about the financing of the project company which is formed to operate the PPP, rather than the revenue or charges expected over the course of the contract.

Reply:
Ah, yes, that was what I was looking for; I had skipped the contract implementation extensions, as I had assumed they only covered actuals, not estimates.

Reply:
I've written guidance that relies on the Charges extension.